ci/integration_tests/src/tests/cw20_stake_test.rs

```diff
@@ -95,20 +96,20 @@ fn execute_stake_tokens(chain: &mut Chain) {
 
     assert_eq!(staked_value.value, Uint128::new(100));
 
-    // TODO: Once cosm-orc supports `poll_for_n_blocks(1)` uncomment below:
-    //
-    // let res = chain
-    //     .orc
-    //     .query(
-    //         "cw_core",
-    //         "exc_stake_q_power",
-    //         &cw_core::msg::QueryMsg::VotingPowerAtHeight {
-    //             address: chain.user.addr.clone(),
-    //             height: None,
-    //         },
-    //     )
-    //     .unwrap();
-    // let power: VotingPowerAtHeightResponse = res.data().unwrap();
-    //
-    // assert_eq!(power.power, Uint128::new(100));
+    chain.orc.poll_for_n_blocks(1, 20_000).unwrap();
```

Comment:
This will block the current thread until we process `1` block, erroring if we wait more than 20 seconds.

Reply:
such a cool api!!